workloads/opea/chatqna/README.md

```diff
@@ -0,0 +1,101 @@
+# Deploy OPEA ChatQnA workload on OCP
+
+## Overview
+The workload is based on the [OPEA ChatQnA Application](https://github.com/opea-project/GenAIExamples/tree/v0.8/ChatQnA) running on Intel® Gaudi Accelerator with OpenShift and OpenShift AI. Refer to the [OPEA Generative AI Examples](https://github.com/opea-project/GenAIExamples/tree/v0.8) for more details about the OPEA workloads. 
+ 
+## Prerequisites
+* Provisioned RHOCP cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster)
+* The Persistent storage using NFS is ready. Refer to [documentation](https://docs.openshift.com/container-platform/4.16/storage/persistent_storage/persistent-storage-nfs.html) for the details to set it up.
+
+**Note**: Refer to [documentation](https://docs.openshift.com/container-platform/4.16/storage/index.html) for setting up other types of persistent storages.
+* Provisioned Intel Gaudi accelerator on RHOCP cluster. Follow steps [../../../gaudi/README.md]
+* Installed and Configured RHOAI. Follow steps [here](../inference/README.md/#install-rhoai) 
+* Minio based S3 service ready for RHOAI. Follow steps [here](https://ai-on-openshift.io/tools-and-applications/minio/minio/#create-a-matching-data-connection-for-minio)
+* Create Persistent Volumes
+
+First, create namespace: ```oc create namespace opea-chatqna```.
+
+The NFS is used to create the Persistent Volumes for ChatQnA MegaService to claim and use.
+
+Make sure to update NFS server IP and path in ```persistent_volumes.yaml``` before applying command below.
+For example:
+```
+  nfs:
+    server: 0.0.0.0 # nfs server
+    path: /my_nfs # nfs path
+```
+
+1. From CLI
+  
+**Note**: The NFS is used to provide the Persistent Volume as reference here. 
+
+``` 
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/workloads/opea/chatqna/persistent_volumes.yaml
+
+```
+``` 
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/workloads/opea/chatqna/persistent_volume_claims.yaml
+
+```
+* Check persistent volume claims are bound to the volumes:
+```
+NAME                          STATUS   VOLUME                       CAPACITY   ACCESS MODES   
+chatqna-megaservice-pvc-0   Bound    chatqna-megaservice-pv-0   100Mi      RWO            
+chatqna-megaservice-pvc-1   Bound    chatqna-megaservice-pv-1   100Mi      RWO            
+chatqna-megaservice-pvc-2   Bound    chatqna-megaservice-pv-2   100Mi      RWO            
+```
+2. From Web Console
+
+Navigate to ```Storage``` -> ```PersistentVolumes```. Click on ```CreatePersistentVolume``` and copy the above yaml. Repeat the same for ```PersistentVolumeClaims```.
+
+### ChatQnA backend- langchain and redis vector database
+1. Deploy anyuid_rbac required for redis vector db and chatqna megaservice
+
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/anyuid_rbac.yaml
```

Comment:
Can we merge this anyuid_rbac.yaml with redis_deployment_service.yaml and remove this step to make it simple?